.github/workflows/push.yaml

```diff
@@ -125,3 +132,30 @@ jobs:
         run: |
           export TAG=pr-${{ github.event.pull_request.number }}-${PR_SHA:0:8}
           ci/toolchain_manager.sh publish "${TAG}"
+
+  package:
+    permissions:
+      contents: write
+    name: Create draft release
+    needs: build-workspaces
+    if: ${{ github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v4
+
+      - name: Package manifests for server
+        run: make -C server package
+
+      - name: Package manifests for operator
+        run: make -C operator package
+
+      - name: Make a draft release
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "operator/operator.tar.gz,server/server.tar.gz"
```

Comment:
shouldn't they be in `operator/out/operator.tar.gz` and `server/out/server.tar.gz`?

Reply:
Fixed in 1177ea0.